engines/cge/detection.cpp

```diff
@@ -111,6 +112,18 @@ static const ADExtraGuiOptionsMap optionsList[] = {
 		}
 	},
 
+#ifdef USE_TTS
+	{
+		GAMEOPTION_TTS,
+		{
+			_s("Enable Text to Speech"),
+			_s("Use TTS to read the descriptions (if TTS is available)"),
```

Comment:
Is this tooltip accurate? I would think it reads more than just the descriptions (such as the dialogs as well)?